spec/jobs/redeliver_web_hook_event_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "excon"
+
+RSpec.describe Jobs::RedeliverWebHookEvent do
+  subject(:job) { described_class.new }
+
+  fab!(:web_hook)
+  fab!(:post)
+
+  REDELIVERED = "redelivered"
```

Comment:
No need to redefine this here, you should be able to call `Jobs::RedeliverWebHookEvent::REDELIVERED`